### PR TITLE
firefox-esr: update to 60.5.0.

### PR DESCRIPTION
**Pull request by Johnnynator**
Continuation of #3516 Builded for x86_64{,-musl} and i686, Aarch64{,-musl} amd armv7l{,-musl}

Todo: Testing

Test repo: http://144.76.1.220/void/firefox-esr/
Key Fingerprint: `50:57:0d:82:cc:fa:c0:4e:5d:32:ab:fb:1d:4a:8f:cb`

### CODE REVIEW

**Comment by Johnnynator** [comment]
@q66 Can you take a look at ppc.

**Comment by q66** [comment]
Can't take a look until next week as I'm in Belgium for FOSDEM now. However, at least one thing I can see, `--disable-elf-hack` needs to be disabled on ppc64 targets, so drop it from the case, I think it should only be added for `x86_64*|i686*|arm*`, so not aarch64 either.

**Comment by Johnnynator** [comment]
> Can't take a look until next week as I'm in Belgium for FOSDEM now. However, at least one thing I can see, `--disable-elf-hack` needs to be disabled on ppc64 targets, so drop it from the case, I think it should only be added for `x86_64*|i686*|arm*`, so not aarch64 either.

Ok, I already did remove it for aarch64, i686 seems to work without adding it, will check it later.

File: srcpkgs/firefox-esr/template [hunk 1]
```diff
@@ -3,47 +3,43 @@
...
-build_options_default="alsa dbus gtk3 pulseaudio startup_notification xscreensaver"
-desc_option_rust="Build rust components"
+build_options="alsa dbus gtk3 pulseaudio startup_notification xscreensaver sndio"
+build_options_default="alsa dbus gtk3 pulseaudio startup_notification xscreensaver sndio"
 
-case "$XBPS_TARGET_MACHINE" in
-	i686|x86_64*) build_options_default+=" rust";;
+case $XBPS_TARGET_MACHINE in
+	armv6*)
+		broken="required NEON extensions are not suppored on armv6"
```
Comment (jnbr) [inline srcpkgs/firefox-esr/template]: typo (suppored -> supported)
I wil lfix it in the firefox template next time I toch it

Comment (Johnnynator) [reply srcpkgs/firefox-esr/template]: Fixed it.

**Pull request merged**